marbot-reserved-instance.yml

```diff
@@ -0,0 +1,404 @@
+---
```

Comment:
Why did you split this up into a standalone stack? Because not everyone is using RIs?

Reply:
yes. I guess Savings Plans will replace RIs?